### PR TITLE
Release changes for v1.15.0-rc1 release 

### PR DESCRIPTION
**Pull request by sanketsudake**
Signed-off-by: Sanket Sudake <sanketsudake@gmail.com>

### CODE REVIEW

File: docs/content/en/docs/installation/compatibility.md [hunk 1]
```diff
@@ -5,14 +5,29 @@ description: >
...
-| 1.11.2          | 1.5          | v0.1                          |
-| 1.12.0          | 2.0          | v0.5                          |
-| 1.13.1          | 2.0          | v0.6 (except GCP pub-sub v0.1)|
-| 1.14.0          | 2.0          | v0.7 (except GCP pub-sub v0.2)|
+| Fission version | Keda version | Keda Connectors image tag      |
+| --------------- | ------------ | ------------------------------ |
+| 1.11.2          | 1.5          | v0.1                           |
+| 1.12.0          | 2.0          | v0.5                           |
+| 1.13.1          | 2.0          | v0.6 (except GCP pub-sub v0.1) |
+| 1.14.0          | 2.0          | v0.7 (except GCP pub-sub v0.2) |
```
Comment (vishal-biyani) [inline docs/content/en/docs/installation/compatibility.md:13]: I think we should call it 2.0 + so people don't stick to just 2.0 version itself!

File: docs/content/en/docs/releases/v1.15.0-rc1.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+### Internal Fixes
+
+- Updated Fission base image from Alpine 3.13 to Alpine 3.14. [PR #2160](https://github.com/fission/fission/pull/2160)
+- Updated Fission to use Go 1.16 and Kubernetes dependencies to 1.21 [PR #2168](https://github.com/fission/fission/pull/2168)
+- Stow client updated v0.2.7 [PR #2195](https://github.com/fission/fission/pull/2195)
+- Using goreleaser for faster buids and releases [PR #2189](https://github.com/fission/fission/pull/2189)
+
+## Note
+
+- From 1.15 release, Fission would be have having one tag only of format v1.15.x etc. instead of two tags earlier. Please ensure you are using v1.15.0 in helm install and not 1.15.0.
```
Comment (vishal-biyani) [inline docs/content/en/docs/releases/v1.15.0-rc1.md:66]: Sorry what does this mean?

**Review by vishal-biyani — COMMENTED** [review]
Two small comments

File: docs/content/en/docs/releases/v1.15.0-rc1.md (hunk 2 above)
Comment (sanketsudake) [reply docs/content/en/docs/releases/v1.15.0-rc1.md:66]: Added example.

Comment (sanketsudake) [reply docs/content/en/docs/releases/v1.15.0-rc1.md:66]: Helm smart works with both version strings though.

```
-> % helm show values --version 1.15.0-rc1   fission-charts/fission-core | grep imageT
ag
imageTag: v1.15.0-rc1
  imageTag: v1.15.0-rc1
-> % helm show values --version v1.15.0-rc1   fission-charts/fission-core | grep imageTag
imageTag: v1.15.0-rc1
  imageTag: v1.15.0-rc1
```

**Pull request merged**